pkg/cli/prompt/text/text_test.go

```diff
@@ -78,11 +78,11 @@ func Test_E2E(t *testing.T) {
 			Placeholder: "test placeholder",
 		}
 		model := NewTextModel("test prompt", options)
-		return teatest.NewTestModel(t, model, teatest.WithInitialTermSize(80, 50))
+		return teatest.NewTestModel(t, model, teatest.WithInitialTermSize(18, 50))
 	}
 
 	normalizeOutput := func(bts []byte) string {
-		return stripansi.Strip(strings.ReplaceAll(string(bts), "\r\n", "\n"))
+		return stripansi.Strip(strings.ReplaceAll(string(bts), "\r", ""))
```

Comment:
Please check this update: https://github.com/charmbracelet/bubbletea/pull/1008/files#diff-eec1160b5c25625c859865f3ea86e45e76521a2dd877c78a508183b968646f7dR235.